.github/workflows/promptflow-sdk-cli-test.yml

```diff
@@ -95,20 +95,35 @@ jobs:
       working-directory: ${{ env.testWorkingDirectory }}
       run: |
         cp ${{ github.workspace }}/src/promptflow/dev-connections.json.example ${{ github.workspace }}/src/promptflow/connections.json
-    - name: Get number of CPU cores
-      uses: SimenB/github-actions-cpu-cores@v1
-      id: cpu-cores
-    - name: Run Test
+    - name: Run SDK CLI Test
       shell: pwsh
       working-directory: ${{ env.testWorkingDirectory }}
       run: |
-        gci env:* | sort-object name
         python "../../scripts/building/run_coverage_tests.py" `
-        -p promptflow `
-        -t ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test `
-        -l eastus `
-        -m "unittest or e2etest" `
-        --coverage-config ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test/.coveragerc
+          -p promptflow `
+          -t ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test `
+          -l eastus `
+          -m "unittest or e2etest" `
+          --coverage-config ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test/.coveragerc `
+          -o "${{ github.workspace }}/test-results-sdk-cli.xml" `
+          --ignore-glob ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test/e2etests/test_executable.py
+    - name: Install pf executable
+      shell: pwsh
+      working-directory: artifacts
+      run: |
+        Set-PSDebug -Trace 1
+        gci ./promptflow -Recurse | % {if ($_.Name.Contains('.whl')) {python -m pip install "$($_.FullName)[executable]" --force}}
+        pip freeze
+    - name: Run SDK CLI Executable Test
+      shell: pwsh
+      working-directory: ${{ env.testWorkingDirectory }}
+      run: |
+        python "../../scripts/building/run_coverage_tests.py" `
+          -p promptflow `
+          -t ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test/e2etests/test_executable.py `
+          -l eastus `
+          -m "unittest or e2etest" `
+          -o "${{ github.workspace }}/test-results-sdk-cli-executable.xml"
```

Comment:
where is this `test-results-sdk-cli-executable.xml`

Reply:
![image](https://github.com/microsoft/promptflow/assets/2208599/23d799fe-c871-4a41-89ae-0c7c4f40b3d4)
uploaded as artifact.

Reply:
Good catch, this is uploaded but not analyzed. since there is only 1 test in executable test, and this executable test could raise error by itself, I'll include it in the published result in my next patch.